appcenter-analytics/package.json

```diff
@@ -43,9 +43,7 @@
   },
   "jest": {
     "preset": "react-native",
-    "setupFilesAfterEnv": [
-      "./jest/setup.js"
-    ],
+    "setupTestFrameworkScriptFile": "./jest/setup.js",
```

Comment:
Looks like [setupTestFrameworkScriptFile is deprecated in favor of setupFilesAfterEnv](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array).

Reply:
@dhei `setupFilesAfterEnv` doesn't work with our current setup. I know @jackycodes was looking at it a while ago, I don't think he ever got it working.

Reply:
Do we know when jest will deprecate `setupTestFrameworkScriptFile`?

Reply:
@dhei no, but it looks like jest@^24.x.x fixes the issue. I'll upgrade jest instead - that's better.